design/release-notes/release-0.11/draft-release-notes.md

```diff
@@ -60,8 +60,22 @@ A table of resources and their old and new `apiVersion`s:
 You must also make sure to update all references to cert-manager in annotations to their
 new prefix:
 
-| Annotation | Affected resources | New annotation |
-|------------|--------------------|----------------|
+| Annotation                                   | Affected resources | New annotation                              |
+|----------------------------------------------|--------------------|---------------------------------------------|
+| certmanager.k8s.io/acme-http01-edit-in-place | Ingress            | acme.cert-manager.io/http01-edit-in-place   |
+| certmanager.k8s.io/acme-http01-ingress-class | Ingress            | acme.cert-manager.io/http01-ingress-class   |
+| certmanager.k8s.io/issuer                    | Ingress            | cert-manager.io/issuer                      |
+| certmanager.k8s.io/cluster-issuer            | Ingress            | cert-manager.io/cluster-issuer              |
+| certmanager.k8s.io/acme-challenge-type       | Ingress            | DEPRECATED                                  |
+| certmanager.k8s.io/acme-dns01-provider       | Ingress            | DEPRECATED                                  |
+| certmanager.k8s.io/alt-names                 | Ingress            | cert-manager.io/alt-names                   |
+| certmanager.k8s.io/ip-sans                   | Ingress            | cert-manager.io/ip-sans                     |
```

Comment:
`certmanager.k8s.io/alt-names` and `ip-sans` are on *Secret* resources, along with a number of other annotations:

```
      cert-manager.io/alt-names: example.com
      cert-manager.io/certificate-name: example-com-tls
      cert-manager.io/common-name: example.com
      cert-manager.io/ip-sans: ""
      cert-manager.io/issuer-kind: ClusterIssuer
      cert-manager.io/issuer-name: letsencrypt-prod
      cert-manager.io/uri-sans: ""
      certmanager.k8s.io/alt-names: example.com
      certmanager.k8s.io/common-name: example.com
      certmanager.k8s.io/ip-sans: ""
      certmanager.k8s.io/issuer-kind: ClusterIssuer
      certmanager.k8s.io/issuer-name: letsencrypt-prod
```